ui/component/core/src/util.ts

```diff
@@ -274,6 +275,22 @@ export function objectsEqual(obj1?: any, obj2?: any, deep: boolean = true): bool
 
     return false;
 }
+/**
+ * Deep diff between two object, using lodash
+ * @param  {Object} object Object compared
+ * @param  {Object} base   Object to compare with
+ * @return {Object}        Return a new object who represent the diff
+ */
+export function difference(object?: any, base?: any): any {
+    function changes(object?: any, base?: any) {
```

Comment:
Prefer arrow functions when inlining functions

Reply:
Sure! I adjusted it in the newest commit.